sklearn/datasets/openml.py

```diff
@@ -489,24 +589,41 @@ def fetch_openml(name=None, version='active', data_id=None, data_home=None,
         If True, returns ``(data, target)`` instead of a Bunch object. See
         below for more information about the `data` and `target` objects.
 
+    return_frame : boolean, default=False
+        If True, returns a Bunch where the data attribute is a pandas
+        DataFrame.
+
+    chunksize : int, default=5000
+        Number of rows to read at a time when constructing a dataframe.
```

Comment:
Is it possible to estimate against working_memory instead?

Reply:
It is possible. We would need to estimate the amount of memory used per row.